src/renderer/App.js

```diff
@@ -352,7 +352,8 @@ export default defineComponent({
     },
 
     handleKeyboardShortcuts: function (event) {
-      if (event.shiftKey && event.key === '?') {
+      // ignore user typing in HTML `input` elements
+      if (event.shiftKey && event.key === '?' && document.activeElement?.tagName !== 'INPUT') {
```

Comment:
Any reason to use `document.activeElement` instead of `event.target`?

Reply:
Nope. That looks prettier, so I'll go with that